README.md

```diff
@@ -12,16 +12,43 @@ Creating what we call [starter issues](http://hood.ie/blog/starter-issues.html)
 
 Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the issue directly in the master branch or creating a pull request which is time-consuming, I can simply create a new branch that is called something like **first-timers-only-typo-in-title.** GitHub will then notify the "first-timers-only issue bot" about the new branch using Webhooks. The bot is listening to any new branch starting with **first-timers-only** and it will create a new issue on your repo. It is currently setup to the Hoodie repo with a template and assigns the first-timers-only and up-for-grabs labels but can be modified by the developer. The commit body can be used to add some context information and if left empty, the 🤔 **What you will need to know** section of the issue will simply say "Nothing :)".
 
-### 🕜 💻 Setup
+
+### 😮 🙌 👀 🎉 Use Our Bot!
+![Install App](https://github.com/apps/first-timers-bot)
+
+Try it out on this ![repo](https://github.com/First-timers-bot/first-timers-test)!
```

Comment:
unfortunately, that will not work, because people don’t have access to this repository until you invite them as collaborator. Until then they will fork, github does it for them. They cannot create a new branch themselves. Give it a try on a repo you that you are not a collaborator on

Reply:
@gr2m Oh true! Thanks for catching that